examples/AddressBook/models/cpp/AddressBookCpp.h

```diff
@@ -58,7 +58,7 @@ class AddressBookImplementation : public AddressBook
 
     void createNewContact() override
     {
-        qCDebug(LogGeneral) << "C++ createNewContact called";
```

Comment:
why categorized logs are removed from examples?

Reply:
Facelift categories are not meant to be used by application code. Our examples should be written accordingly.